judges/cuda/test.yaml

```diff
@@ -16,6 +16,7 @@ kolejka:
         storage     : '2G'
         workspace   : '2G'
         gpus        : 1
+        memory_gpu  : '128MB'
```

Comment:
This should be larger. Judge default is 1G, example is 256MB.

Reply:
Sure. It's part of debugging artifacts. I wasn't sure about the source of values in Task limit. It was done just to made them odd.